specs/userService.yaml

```diff
@@ -144,34 +87,62 @@ userSettingsEndpoint:
 adminEndpoints:
   get:
     summary: Get all users.
-    description: Get all users.
-    operationId: admin-get-user
+    description: |-
+      This endpoint returns all users with settings stored.
+
+      The endpoint is only accessible by administrators and considered **unstable**.
+    operationId: user:admin:getAll
+    deprecated: true
```

Comment:
Why the deprecation?

Reply:
> The endpoint is only accessible by administrators and considered **unstable**.

Those endpoints gonna change as we don't have yet a final concept on access control for administrators (offer no scientific research opportunity for our current focus. And furthermore you are not supposed to use it anyway :)